### ISSUE TITLE
Create specific page-review-template for region managers

### ISSUE THREAD

**Issue by timobrembeck** [digitalfabrik/integreat-cms#1447]
### Motivation

When users without publishing rights save pages as "in review", the other users who review that page should have a direct possibility to see what the user changed

### Proposed Solution

Create a new template for the page form which opens when the status is "in review" and the current user has the permission to review/publish the page.
In this template, the user should see the diff from the last version that was not "in review" to the latest version.
Then, there should be two buttons "accept changes" or "reject changes" which should either convert the status to "public" or "draft" (but not all past versions, just the versions that were marked as "in review").

### Additional Context

Optionally, we could highlight all pages that require a review in the page list for users that do have the permission to publish them.

**Comment by MizukiTemma** [comment]
FYI: The version view (changes are highlighted) and the row "Pages pending approval" in the dashboard may cover this issue 👀

**Comment by laravoelk** [comment]
I agree with Mizuki. I also think expecially "Pages pending approval" cover this.

**Comment by JoeyStk** [comment]
Thank you a lot for your insights! Then we would close this issue but keep it as an idea :)
